src/components/RightPane/CoursePane/SearchForm/TermSelector.tsx

```diff
@@ -16,17 +21,17 @@ class TermSelector extends PureComponent {
     };
 
     componentDidMount = () => {
-        document.addEventListener('keydown', this.enterEvent, false);
+        // document.addEventListener('keydown', this.enterEvent, false);
```

Comment:
Why is this being commented out?

Reply:
`TermSelector` never had a `enterEvent` listener. I suspect this was probably copypasta'd from another component in the before-times, and nobody caught it because JavaScript lets you do anything with things that don't exist. If it would make more sense to simply remove that line, I'm fine with it.

Reply:
Yeah, just remove it.